ras_party/controllers/account_controller.py

```diff
@@ -58,6 +58,7 @@
 
 NO_RESPONDENT_FOR_PARTY_ID = "There is no respondent with that party ID"
 EMAIL_VERIFICATION_SENT = "A new verification email has been sent"
+ACCOUNT_EMAIL_CHANGE_VERIFICATION_SENT = "A new account email change verification email has been sent"
```

Comment:
This reads kinda weird.... maybe
`A verification email has been sent to change the email address on the account` or something like that?  My suggestion kinda sucks so don't use that!  Can you ask for possibly better wording for it from someone else?

Reply:
👍